routes/auth.js

```diff
@@ -18,6 +18,13 @@ module.exports = sequelize => {
     }
   });
 
+  // Executes the getAdmin function to retrieve the value in Admin column of a given User
+  router.post('/getAdmin', function (req, res, next) {
```

Comment:
this is a get function, so the request should be 'router.get' instead of 'router.post'